### ISSUE TITLE
🐛 Bug Report: Cannot read properties of undefined (reading 'isMultiSelect')

### ISSUE THREAD

**Issue by IlyaSavich** [backstage/backstage#17531]
### 📜 Description

Hi team, I noticed that for after upgrade Backstage to `v1.13.1` for one plugin `"@backstage/plugin-home": "^0.5.0"` I constantly get error
```
Cannot read properties of undefined (reading 'isMultiSelect')
```
for any case where I just used any import from this plugin. I didn't find anything related to this issue in codebase, so it looks a lot like a bug with some dependency.

Could someone take a look ?

### 👍 Expected behavior

App works

### 👎 Actual Behavior with Screenshots

Get error on page load `Cannot read properties of undefined (reading 'isMultiSelect')`

### 👟 Reproduction steps

1. Upgrade `@backstage/plugin-home` to `^0.5.0`
2. Build
3. Open Backstage app

### 🖥️ Your Environment

OS:   Darwin 21.6.0 - darwin/arm64
node: v16.19.0
yarn: 3.3.1
cli:  0.22.6 (installed)
backstage:  1.13.1

Dependencies:
  @backstage/app-defaults                          1.3.0
  @backstage/backend-app-api                       0.4.2
  @backstage/backend-common                        0.18.4
  @backstage/backend-dev-utils                     0.1.1
  @backstage/backend-plugin-api                    0.5.1
  @backstage/backend-tasks                         0.5.1
  @backstage/backend-test-utils                    0.1.36
  @backstage/catalog-client                        1.4.1
  @backstage/catalog-model                         1.3.0
  @backstage/cli-common                            0.1.12
  @backstage/cli-node                              0.1.0
  @backstage/cli                                   0.22.6
  @backstage/config-loader                         1.2.0
  @backstage/config                                1.0.7
  @backstage/core-app-api                          1.7.0
  @backstage/core-components                       0.12.5, 0.13.0
  @backstage/core-plugin-api                       1.5.1
  @backstage/dev-utils                             1.0.14
  @backstage/errors                                1.1.5
  @backstage/eslint-plugin                         0.1.3
  @backstage/integration-aws-node                  0.1.2
  @backstage/integration-react                     1.1.12
  @backstage/integration                           1.4.4
  @backstage/plugin-adr-common                     0.2.8
  @backstage/plugin-adr                            0.5.0
  @backstage/plugin-api-docs                       0.9.2
  @backstage/plugin-app-backend                    0.3.44
  @backstage/plugin-auth-backend                   0.18.2
  @backstage/plugin-auth-node                      0.2.13
  @backstage/plugin-catalog-backend-module-ldap    0.5.11
  @backstage/plugin-catalog-backend-module-openapi 0.1.10
  @backstage/plugin-catalog-backend                1.9.0
  @backstage/plugin-catalog-common                 1.0.13
  @backstage/plugin-catalog-graph                  0.2.29
  @backstage/plugin-catalog-node                   1.3.5
  @backstage/plugin-catalog-react                  1.5.0
  @backstage/plugin-catalog                        1.10.0
  @backstage/plugin-cost-insights-common           0.1.1
  @backstage/plugin-cost-insights                  0.12.6
  @backstage/plugin-gcalendar                      0.3.13
  @backstage/plugin-home                           0.5.0
  @backstage/plugin-kubernetes-backend             0.10.0
  @backstage/plugin-kubernetes-common              0.6.2
  @backstage/plugin-kubernetes                     0.8.0
  @backstage/plugin-org                            0.6.7
  @backstage/plugin-permission-common              0.7.5
  @backstage/plugin-permission-node                0.7.7
  @backstage/plugin-permission-react               0.4.12
  @backstage/plugin-proxy-backend                  0.2.38
  @backstage/plugin-scaffolder-backend             1.13.1
  @backstage/plugin-scaffolder-common              1.2.7
  @backstage/plugin-scaffolder-node                0.1.2
  @backstage/plugin-scaffolder-react               1.3.0
  @backstage/plugin-scaffolder                     1.13.0
  @backstage/plugin-search-backend-module-catalog  0.1.0
  @backstage/plugin-search-backend-node            1.2.0
  @backstage/plugin-search-backend                 1.3.0
  @backstage/plugin-search-common                  1.2.3
  @backstage/plugin-search-react                   1.5.2
  @backstage/plugin-search                         1.2.0
  @backstage/plugin-tech-insights-backend          0.5.10
  @backstage/plugin-tech-insights-common           0.2.10
  @backstage/plugin-tech-insights-node             0.4.2
  @backstage/plugin-tech-insights                  0.3.9
  @backstage/plugin-tech-radar                     0.6.3
  @backstage/plugin-user-settings                  0.7.2
  @backstage/release-manifests                     0.0.9
  @backstage/test-utils                            1.3.0
  @backstage/theme                                 0.2.19
  @backstage/types                                 1.0.2
  @backstage/version-bridge                        1.0.4

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, I don't have time to work on this right now

**Comment by benjdlambert** [comment]
Fixed by #17532

**Comment by benjdlambert** [comment]
This has been released in 1.13.2.
